baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferAssertj.java

```diff
@@ -286,42 +289,61 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
         if (ASSERT_EQUALS_FLOATING.matches(tree, state)) {
             return withAssertThat(tree, state, (assertThat, fix) -> fix
                     .addStaticImport("org.assertj.core.api.Assertions.within")
-                    .replace(tree, assertThat + "(" + argSource(tree, state, 1) + ").isCloseTo("
-                            + argSource(tree, state, 0) + ", within(" + argSource(tree, state, 2) + "))"));
+                    .replace(tree, assertThat + "(" + argSource(tree, state, 1)
```

Comment:
I think these would be much easier to read as format strings

Reply:
Good call, updated.